bin/templates/templates.xml

```diff
@@ -1,22 +1,21 @@
 <?xml version="1.0"?>
 <!--
-  ~ Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
We have configured autostyle for our sources and these changes were made a few days ago in trunk. Please run `./gradlew style` before opening the PR.

Generally look out for whitespace changes, that you didn't intend. Don't commit those, it makes it harder to reason about the real changes.